app/files/views/file.py

```diff
@@ -0,0 +1,84 @@
+from django.http import Http404
+from rest_framework import status
+from rest_framework.response import Response
+
+from app.common.permissions import BasicViewPermission
+from app.common.viewsets import BaseViewSet
+from app.files.models.file import File
+from app.files.serializers.file import (
+    CreateFileSerializer,
+    FileSerializer,
+    UpdateFileSerializer,
+)
+
+
+class FileViewSet(BaseViewSet):
+    serializer_class = FileSerializer
+    permission_classes = [BasicViewPermission]
+    queryset = File.objects.all()
```

Comment:
For å optimalisere kan du bruke select_related() der du sender inn user eller gallery. Sjekk Event viewsettet og dokumentasjon: https://docs.djangoproject.com/en/5.1/ref/models/querysets/#select-related

dette er for å optimalisere joins i SQL spørringen